kerdock.yml

```diff
@@ -0,0 +1,51 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# code id, physical, logical are all lower case
+code_id: kerdock
+physical: bits
+logical: bits
+
+name: 'kerdock'
+introduced: '\url{https://doi.org/10.1016/S0019-9958(72)90376-2}'
+
+description: 'Kerdock code is a \([[2^m, 2^{2m}, 2^{m-1} - 2^{frac{m}{2}-1}]]\) binary code where \(m\) is even. The size of code book is \(|2^{m-1}||R(1,m)|\) where \(R(1,m)\)
+is the corresponding first order Reed Muller code. The code is non linear and can exist for perimeters for which there is no linear code (See Nordstrom-Robinson). The relative
+minimum distance tends to \(\frac{1}{2}\) as \(m\) becomes large.'
+
+#protection: ''
+
+features:
+
+  rate: 'The transmission rate is \(\frac{2m}{2^m}\) which tends to 0 as \(m\) becomes large, hence these codes are asymptotically poor.'
+
+  encoders:
+    - 'Boolean functions are of the form \(Q(X) + l(X) + b\)' where \(Q(X) is \sum_{1 \leq i < j \leq m} a_{ij}X_{i}X_{j}\), \(l(X)\) is linear in \(m\) variables and \(b\) is 
```

Comment:
Please describe the context of these boolean functions. Are they helping determine the codewords? If so, place into description, which needs a way to define the code anyway.